---
#Soft trunk parameters are defined here
robot_name: 2segment
#Physical segments of the arm
num_segments: 2
#PCC sections per segment
sections_per_segment: 1
#Masses of segments and connectors, starting at topmost segment
masses: [0.170, 0.163]
#Lengths of segments and connectors, starting at topmost segment
lengths: [0.14, 0.135]
#Diameters of segment top and bottoms, contains num_seg+1 values
diameters: [0.035, 0.028, 0.0198]
#Angle of arm relative to upright position
armAngle: 180
#Shear moduli of the segments
shear_modulus: [35000,62000]
#Drag coefficients of the segments
drag_coef: [28000,8000]
#Drag coefficients for phi,theta representation
d_vect: [0.001,0.001]
#Stiffness coefs for phi,theta representation (?)
k_vect: [0.12,0.08]
model: augmented
...
